.github/workflows/ci.yaml

```diff
@@ -0,0 +1,30 @@
+name: CI build
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+      - uses: actions/cache@v2
```

Comment:
What's going on with this one? Are we sharing go vendor between runs?

Reply:
yeah caches the packages and build become faster from 2m to 38s.